Jenkinsfile_nightly

```diff
@@ -29,6 +29,7 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 withNightlyPipeline(type, product, component) {
     loadVaultSecrets(secrets)
     enableMutationTest()
-    enableDbMigration()
+    enableFortifyScan()
+    enableDbMigration('ccd')
```

Comment:
let's raise a separate PR for this line and merge it, so that this PR contains only its own changes.